beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.wdl

```diff
@@ -475,25 +489,31 @@ task gatk_baserecalibrator {
         File ref_fasta
         File ref_fai
         Int cpu = 2
-        Int mem = 6
         Int preemptible = 2
         Int max_retries = 0
-        Int additional_disk = 0
+        Int additioinal_memory_mb = 0
+        Int additional_disk_gb = 0
     }
     String output_grp = basename(bam, ".bam") + "_bqsr.grp"
-    Float ref_size = size([ref_fasta, ref_fai, ref_dict], "G")
-    Float dbsnp_size = size([dbsnp_vcf, dbsnp_vcf_index], "G")
-    Int jvm_mem = if mem > 1 then mem - 1 else 1
-    Int disk_space = ceil(size(bam, "G") + ref_size + dbsnp_size) + 20 + additional_disk
+    Float ref_size = size([ref_fasta, ref_fai, ref_dict], "GiB")
+    Float dbsnp_size = size([dbsnp_vcf, dbsnp_vcf_index], "GiB")
+    Int mem = ceil(size(bam, "MiB"))+ 6000 + additioinal_memory_mb
```

Comment:
minor nit.  Add space between ')' and '+'
And correct that misspelling here and everywhere.